marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/builtins/steps/mapping/entity-services/lib.sjs

```diff
@@ -155,7 +156,7 @@ function getRelatedMappings(mapping, related = [mapping]) {
 
 function getTargetEntity(targetEntityType) {
   if (!entitiesByTargetType[targetEntityType]) {
-    let entityModel = getModel(targetEntityType);
+    let entityModel = entityLib.findModelForEntityTypeId(targetEntityType);
```

Comment:
This is the only change to existing code, as it was a trivial refactoring to perform.